media/js/plugincheck/tmpl/plugincheck.ui.tmpl.js

```diff
@@ -4,7 +4,8 @@ var vulnerablePluginsTmpl = '{{#vulnerablePlugins}} <tr><th scope="row">' +
                         '<small>{{{ plugin_detail }}}</small>' +
                         '</th>' +
                         '<td class="status">{{plugin_status}}<div class="plugin_version">{{ plugin_version }}</div></td>' +
-                        '<td class="action"><a href="{{ url }}" class="button button-negative" target="_blank">{{button_update}}</a></td>' +
+                        '<td class="action"><a href="{{ url }}" class="button button-negative" target="_blank">{{button_update}}</a>' +
+                        '<a class="vulnerability-link-txt" href="{{ vulnerability_url }} " target="_blank">{{ vulnerability_link_txt }} »</a></td>' +
```

Comment:
Nit: add a CSS class of `more` here for the double caret, instead of including it in the link text directly.